app/controllers/chargeback_controller.rb

```diff
@@ -338,6 +389,56 @@ def cb_assign_field_changed
     end
   end
 
+  # Add a new tier at the end
+  def cb_tier_add
+    @edit = session[:edit]
+    i = params[:id]
+    ii = i.to_i
```

Comment:
I don't know what id is here... is this a `tier.id`?  Isn't an id column already an integer?